src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,57 @@
+package duke.command;
+
+import duke.exceptions.UnclearInstructionException;
+import duke.task.Event;
+import duke.task.Deadline;
+import duke.task.Task;
+import duke.util.TaskList;
+import duke.task.Todo;
+import duke.util.Ui;
+import duke.util.Storage;
+import java.util.ArrayList;
```

Comment:
Do you want to include a linebreak before this for consistency with your other classes?